src/test/scala/chiselTests/ChiselEnum.scala

```diff
@@ -210,7 +210,7 @@ class CastFromNonLitTester extends BasicTester {
   }
 
   val invalid_values =
-    (1 until (1 << EnumExample.getWidth)).filter(!EnumExample.litValues.map(_.litValue).contains(_)).map(_.U)
+    (1 until (1 << EnumExample.getWidth)).filter(!EnumExample.litValues.map(_.litValue.toInt).contains(_)).map(_.U)
```

Comment:
These needed updating due to `-Xlint:infer-any` now being applied to all projects.